<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>回流焊仿真软件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1,1,1,1">
     <item>
      <widget class="QLabel" name="title">
       <property name="text">
        <string>回流焊仿真软件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="argumentPage">
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modelPage">
       <property name="text">
        <string>模型设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dataProcessPage">
       <property name="text">
        <string>数据处理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitSystem">
       <property name="text">
        <string>退出系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>273</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
